### ISSUE TITLE
AWS Signature V4 incorrect for application/x-www-form-urlencoded requests

### ISSUE THREAD

**Issue by garnold** [a85/POSTMan-Chrome-Extension#4151]
## App Details:

```
Postman for Mac
Version 5.5.2 (5.5.2)
OS X 10.13.3
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: *I'm new to Postman so it's always been there for me*

It appears that Postman is calculating the Signature prior to URL encoding the request body. Our API is protected using AWS Signatures V4, and if any of the form data values include spaces the Signature check fails. If I remove spaces from the values, the Signature matches.

The same request works fine (with spaces) in other REST clients like Insomnia.

**Comment by garnold** [comment]
More on this... I believe Postman is calculating the AWSv4 signature before encoding the form parameters in the POST body. JSON POSTs (`application/json`) work fine, but form data (`application/x-www-form-urlencoded`) with parameter values containing spaces do not.

Can someone from Postman check to see what's going on?

**Comment by FabioGentile** [comment]
I'm facing the same issue using SES api: if I try to use a POST method i get a signature error, while if I try with a GET method (same Authorization parameters) it works.

# App Details
```
Postman for Windows
Version 5.5.2
win32 10.0.16299 / x64
```

**Comment by harryi3t** [comment]
@garnold I will look into this issue also.

**Comment by codenirvana** [comment]
We've released a fix for this on our Canary (`version: 7.1.0-canary01`) channel https://www.getpostman.com/canary

@garnold @FabioGentile Please try it out in the Canary version and let us know if you continue to face the issue.
Will update the thread once we release the fix on our stable channel.

**Comment by codenirvana** [comment]
This has been fixed in the latest Postman app.
